src/cljs/main/broadfcui/common/links.cljs

```diff
@@ -11,6 +11,17 @@
                         attributes)
    contents])
 
+(defn create-download [label url filename]
+  [:a {:href url
+       :download filename}
+   label
+   icons/download-icon])
+
+(defn create-download-from-object [label object filename]
+  (let [payload-blob (js/Blob. (js/Array. object) {:type "text/plain"})
+        payload-object-url (.createObjectURL js/URL payload-blob)]
```

Comment:
I prefer to invoke this as `(js/URL.createObjectURL payload-blob)`.

Reply:
Gets rid of the warning, so I'm for it. Thanks.